docs/EN/5-DailyLifewithAAPS/DailyLife-PUMPS.md

```diff
@@ -0,0 +1,66 @@
+# Daily Life - Pumps
```

Comment:
This information seems most valid for tubed pumps? Is there anything to add for patch pumps?

Reply:
I don't have any expertise with patched pumps - we need someone else to add a section on this (are you able to?).